sites/kit.svelte.dev/src/routes/home/Video.svelte

```diff
@@ -0,0 +1,128 @@
+<script>
+	import testimonials from './testimonials.mp4';
+	import volume_off from './volume-off.svg';
+	import volume_high from './volume-high.svg';
+	import { onMount } from 'svelte';
+
+	/** @type {HTMLVideoElement} */
+	let video;
+	let muted = true;
+	let unused = true;
+
+	let d = 0;
+	let t = 0;
+
+	onMount(() => {
+		// I think this binding is broken in SSR because the video already
+		// has a duration by the time hydration occurs. TODO investigate
+		d = video.duration;
+
+		const observer = new IntersectionObserver(
+			(entries) => {
+				for (const entry of entries) {
+					if (entry.isIntersecting) {
+						video.play();
```

Comment:
I think we should only autoplay the video if [reduced motion is not enabled](https://www.scottohara.me/note/2019/07/12/reduced-motion-video.html)